### ISSUE TITLE
bug: built-in module _typing is reported as a missing module

### ISSUE THREAD

**Issue by Zcraft3793** [microsoft/pyright#11147]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

If you have a question about typing or a behavior that you’re seeing in Pyright (as opposed to a bug report or enhancement request), consider posting to the [Pyright discussion forum](https://github.com/microsoft/pyright/discussions).

**Describe the bug**
A clear and concise description of the behavior you are seeing and the expected behavior along with steps to reproduce it.

**Code or Screenshots**
If possible, provide a minimal, self-contained code sample (surrounded by triple back ticks) to demonstrate the issue. The code should define or import all referenced symbols.

```python
import _typing # <module '_typing' (built-in)>
```

If your code relies on symbols that are imported from a third-party library, include the associated import statements and specify which versions of those libraries you have installed.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension, a language server in another editor, integrated into Pylance, or the command-line tool? Which version?

**Comment by erictraut** [comment]
There is no module `_typing` provided in the stdlib typeshed stubs, so pyright has no knowledge of this module. If you'd like to make the argument that `_typing.pyi` should be included in typeshed, please post a feature request to that project. I think it's unlikely that the typeshed maintainers would add such a stub because this is an undocumented and private module.
